app/reducers/initialState.js

```diff
@@ -10,6 +10,11 @@ export default {
     id: '',
     show: false
   },
+  login: {
```

Comment:
👍

Reply:
You can also call this `auth`. I think I'd prefer it instead of `login`. What do you think?

Reply:
Yes, it could work too.